yuuna/lib/src/pages/implementations/subtitle_options_dialog_page.dart

```diff
@@ -315,6 +332,94 @@ class _SubtitleOptionsDialogPage
+                  style: TextStyle(color: fontColor),
+                  readOnly: true,
+                  decoration: InputDecoration(
+                    floatingLabelBehavior: FloatingLabelBehavior.always,
+                    labelText: t.player_option_font_color,
+                    suffixIcon: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      mainAxisSize: MainAxisSize.min,
+                      children: [
+                        JidoujishoIconButton(
+                          size: 18,
+                          tooltip: t.google_fonts,
+                          onTap: () async {
+                            showColorPicker('Font');
+                          },
+                          icon: Icons.color_lens,
+                        ),
+                        JidoujishoIconButton(
+                          size: 18,
+                          tooltip: t.reset,
+                          onTap: () async {
+                            _fontColorController.text = '';
+                            FocusScope.of(context).unfocus();
+                          },
+                          icon: Icons.undo,
+                        ),
+                        const SizedBox(width: 8),
+                      ],
+                    ),
+                  ),
+                ),
+                const Space.small(),
+                Padding(
+                  padding: Spacing.of(context).insets.onlyTop.small,
+                  child: Text(
+                    t.player_option_font_weight,
+                    style: TextStyle(
+                      fontSize: 12,
+                      color: theme.hintColor,
+                    ),
+                  ),
+                ),
+                JidoujishoDropdown<String>(
+                  options: fontWeights,
+                  initialOption: fontWeights[fontWeightIdx],
+                  generateLabel: (weight) => weight,
+                  onChanged: (weight) {
+                    fontWeightIdx = fontWeights.indexOf(weight ?? 'Normal');
+                    setState(() {});
+                  },
+                ),
+                Container(height: 0.45, color: Colors.black87),
+                const Space.small(),
+                TextField(
+                  controller: _outLineColorController,
+                  readOnly: true,
+                  style: TextStyle(color: outLineColor),
+                  decoration: InputDecoration(
+                    floatingLabelBehavior: FloatingLabelBehavior.always,
+                    labelText: t.player_option_outline_color,
+                    suffixIcon: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      mainAxisSize: MainAxisSize.min,
+                      children: [
+                        JidoujishoIconButton(
+                          size: 18,
+                          tooltip: t.google_fonts,
```

Comment:
When you long-press on an icon button, this should show an appropriate tooltip. Since you've changed the behavior of this button, the tooltip should now be different.